### ISSUE TITLE
Something weird in the config.json file?

### ISSUE THREAD

**Issue by F3PiX** [exercism/ruby#831]
While reading the config.json for the current status of Word Count, I found that it is currently `unlocked_by: Isogram`
That surprised me, because Isogram in my own track is one of the last core exercises, and I already  completed Word Count.  Which I _thought_ used to be `unlocked_by: null`.

There are more exercises that have changed - unintentionally, I suspect. Mostly in the difficulties and the unlocks. And for instance Isogram is now marked as core. 

To illustrate:
Here is Word Count in master:

##### Master
https://github.com/exercism/ruby/blob/558aab9382b6eeed495a3d17b1dee42093e3c2de/config.json#L161-L166
(Isogram is now a core exercise.)

##### Old
And here is it from 2 weeks ago, before the latest changes to the config
https://github.com/exercism/ruby/blob/f52fdd4ee07d83ef7117baaac30ce3173de60c08/config.json#L143-L147
(Isogram is not a core exercise.) 

##### In between
In PR #818 it occurs for the first time
https://github.com/substancelab/ruby/blob/b3bcd64461b17fa1e90de035e42cb24199aa2396/config.json#L151-L156

@insti @koppen Can you confirm if these changes to Word Count and other exercises happened unintentionally? Or did I just miss a major decision?

Related: #829 #827

**Comment by Insti** [comment]
:man_shrugging: v2 exercise ordering.
I suspect (but do not intend to do the work to confirm) that changing exercise unlocks is really buggy on the Rails side. So what you see if you had an account before the change is different from what a fresh user would see today.

**Comment by F3PiX** [comment]
First of all: part of this has been solved with merging #827 At least the unlocking of WordCount.

@hilary  Does that explain that the  difficulties have changed? And also which exercises are core? I have no idea what happened before I got involved.
 @insti I think that is not what this issue is about, it's not about the difference before and after, but merely the current state, so for fresh users. Like that Isogram is a core exercise now. 

I am sorry if this is confusing; I am trying to find out what was intended, because in that case I'll leave it alone. And what was unintented, and thus okay for me to work on.
